### DISCUSSION TITLE
iOS 15 / SwiftUI (Swift-Supabase Client) auth crash

### DISCUSSION THREAD

**Discussion by JeevanThandi** [supabase/supabase#3820] [Questions] [answered]
Has anyone come across this issue when using the Swift Supabase client to authenticate? I'm working on a new SwiftUI application using Supabase client and it seems that GoTrue Swift uses [unowned self] in a completion block that causes the supabase client to crash when logging in: 

I logged the issue here:
https://github.com/supabase-community/supabase-swift/issues/13

Hopefully this post will get more eyes on the issue incase anyone else has come across this or something I'm doing wrong.

**Comment by J0** [comment, marked as answer]
Hey! Seems like the linked issue has been closed so I'm going to mark this issue as resolved.

Please feel free to add on if there are further issues though!
